google/cloud/storage/internal/grpc_client.cc

```diff
@@ -293,7 +295,7 @@ StatusOr<storage::BucketMetadata> GrpcClient::CreateBucket(
   grpc::ClientContext context;
   ApplyQueryParameters(context, request);
   auto response = stub_->CreateBucket(context, proto);
-  if (response) return FromProto(*response);
+  if (response) return FromProto(*response, internal::CurrentOptions());
```

Comment:
you might want a test that the `CurrentOptions()` are used, not just any old `Options{}`.

Reply:
Ack.  I will leave as-is for now.